webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin'); // eslint-disable-line
+
+module.exports = {
```

Comment:
- [ ]  Try to adjust you pop up so that user can see everything on your card, such the name of the movies, number of comment and likes etc. It was perfect on mobile view, but on laptop I couldn't see them well. Or maybe you can consider the option of using scroll to pop, that can fix that issue as well. Apply that to make your project more better and clean.